src/devices/bus/vme/vme_hcpu30.cpp

```diff
@@ -0,0 +1,196 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergey Svishchev
+
+/*
+ * Besta HCPU30 board, skeleton driver.
+ *
+ * Supported by SysV R3 "Bestix" port and also by Linux port,
+ * see https://github.com/shattered/linux-m68k
+ *
+ * 68030 @ 33 MHz - primary CPU
+ * 68882 @ ?? Mhz - FPU
+ * 68020 @ ?? MHz - I/O CPU (using shared memory region)
+ *
+ * 4 or 16 MB of DRAM
+ * 8 or 32 KB of NVRAM
+ *
+ * WD33C93 - SCSI
+ * DP8473 - Floppy
+ * i82586? - Ethernet
+ * 62421 - Real-time clock
+ */
+
+#include "emu.h"
+#include "vme_hcpu30.h"
+
+#define LOG_GENERAL 0x01
```

Comment:
LOG_GENERAL is predefined in logmacro.h so no need to duplicate

Reply:
done and done